src/current/v24.2/child-metrics.md

```diff
@@ -0,0 +1,390 @@
+# TYPE sysbytes gauge
+sysbytes{store="1",node_id="1"} 41979
+sysbytes{store="1",node_id="1",tenant_id="3"} 278
+sysbytes{store="1",node_id="1",tenant_id="4"} 225
+sysbytes{store="1",node_id="1",tenant_id="system"} 41476
+~~~
+
+{% assign feature = "virtual" %}
+{% include {{ page.version.version }}/child-metrics-table.md %}
+
+## Clusters with changefeeds
+
+When child metrics is enabled and [changefeeds with metrics labels]({% link {{ page.version.version }}/monitor-and-debug-changefeeds.md %}#using-changefeed-metrics-labels) are created on the cluster, the `changefeed.*` metrics are exported per changefeed metric label with a label for `scope`. The `scope` label may have the values set using the `metrics_label` option. The cardinality increases with the number of changefeed metric labels. An aggregated metric is also included.
+
+For example:
+
+~~~
+ HELP changefeed_error_retries Total retryable errors encountered by all changefeeds
+# TYPE changefeed_error_retries counter
+changefeed_error_retries{node_id="1"} 0
+changefeed_error_retries{node_id="1",scope="default"} 0
+changefeed_error_retries{node_id="1",scope="employees"} 0
+changefeed_error_retries{node_id="1",scope="office_dogs"} 0
+~~~
+
+{% assign feature = "changefeed" %}
+{% include {{ page.version.version }}/child-metrics-table.md %}
+
+## Clusters with row-level TTL jobs
+
+When child metrics is enabled and [row-level TTL jobs]({% link {{ page.version.version }}/row-level-ttl.md %}) are created on the cluster with the [`ttl_label_metrics` storage parameter enabled]({% link {{ page.version.version }}/row-level-ttl.md %}#ttl-metrics), the `jobs.row_level_ttl.*` metrics are exported per TTL job with `ttl_label_metrics` enabled with a label for `relation`. The value of the `relation` label may have the format: `{database}_{schema}_{table}_{primary key}`. The cardinality increases with the number of TTL jobs with `ttl_label_metrics` enabled. An aggregated metric is also included.
+
+For example:
+
+~~~
+# HELP jobs_row_level_ttl_total_rows Approximate number of rows on the TTL table.
+# TYPE jobs_row_level_ttl_total_rows gauge
+jobs_row_level_ttl_total_rows{node_id="1"} 0
+jobs_row_level_ttl_total_rows{node_id="1",relation="default"} 0
+jobs_row_level_ttl_total_rows{node_id="1",relation="defaultdb_public_events__events_pkey_"} 0
+jobs_row_level_ttl_total_rows{node_id="1",relation="defaultdb_public_events_using_date__events_using_date_pkey_"} 0
+~~~
+
+{% assign feature = "row-level-ttl" %}
+{% include {{ page.version.version }}/child-metrics-table.md %}
+
+## Metrics of type histogram
+
+When child metrics is enabled with changefeeds or row-level TTL jobs, be aware that metrics of type HISTOGRAM will increase cardinality quickly. For example, with two changefeeds with metrics labels, the histogram metric `changefeed_flush_hist_nanos` exports child metrics for each bucket for the following `scope` values: `default`,  `employees` and `office_dogs`:
```

Comment:
I think for the changefeed example it might be worth adding a sentence about how we got to those scope values. Otherwise, they kind of come out of nowhere. So something along the lines of: 

> For example, when you create two changefeeds both with the metrics labels `employees` and `office_dogs`, the histogram metric `changefeed_flush_hist_nanos` exports child metrics for each bucket for each metrics label associated to a changefeed and the `default` scope, which includes changefeeds started without a metrics label. Therefore, in this example, `changefeed_flush_hist_nanos` exports child metrics for each bucket for the scope values: `default`, `employees` and `office_dogs`:

Reply:
Good clarification! I updated accordingly.